ros2/test/diagnostics/BUILD.bazel

```diff
@@ -12,27 +11,20 @@ ros2_cpp_binary(
     ],
 )
 
-ros2_bag(
-    name = "bag",
-    idl_deps = [
-        "@ros2_common_interfaces//:diagnostic_msgs",
-        "@ros2_rcl_interfaces//:rcl_interfaces",
-        "@ros2_rosbag2//:rosbag2_interfaces",
-    ],
-)
-
 ros2_test(
     name = "diagnostic_heartbeat_test",
     size = "small",
     data = [
         "aggregator_config.yaml",
-        ":bag",
     ],
     launch_file = "tests.py",
     nodes = [
         ":diagnostic_publisher",
         "@ros2_diagnostics//:aggregator_node",
     ],
-    tags = ["cpu:2"],
```

Comment:
Not sure what this was good for. Can revert if this does anything useful.

Reply:
https://bazel.build/reference/test-encyclopedia: this tag give tests more cores. I put this for robustness, won't harm to have this IMO.

Reply:
At the same time, we don't _need_ it any more. With a timeout of 10 s, a single core is easily able to handle this. I ran it with this on my laptop with 4 physical cores and it passed just fine:
```
bazel test --runs_per_test=100 --jobs=100 --local_cpu_resources=100 //ros2/test/diagnostics:diagnostic_heartbeat_test
```
So would keep it remove, since it adds unnecessary noise. If you insist, of course, I will put it back.